drivers/led/eos_s3_led1_ip.h

```diff
@@ -0,0 +1,1901 @@
+uint32_t	axFPGABitStream[] = {
```

Comment:
> We are planning to have a separate repo to store our IPs and drivers. I'm planning to bring that repo as a sub module under "eos-s3-hal" repo, which is already defined in zephyr manifest file.
> I see `west update` doesn't clone the module repo recursively. I think after `west update`, we have to do `git submodule init` and `git submodule update`. Please suggest if there is a better way.

Do you have any suggestion on this point?

Reply:
I suppose it didn't clone because there was no `zephyr/module.yml` file in repository with IPs and/or there was no entry in `west.yml` in main zephyr repo. If you want to add IP repository as submodule to `hal_quicklogic` please do as follow:
* Add to `west.yml` in your main zephyr repository entry:
```
    - name: <name_of_module>
      remote: QuickLogic-Corp
      repo-path: <repo_name>
      revision: <SHA>
      path: modules/hal/quicklogic/<name_of_module>
```
* Add proper remote in `remotes` section of `west.yml`:
```
    - name: QuickLogic-Corp
      url-base: https://github.com/QuickLogic-Corp
```
* Add proper `CMakeLists.txt` and `zephyr/module.yml` to repository you want to add (take a look at these from [hal_quicklogic](https://github.com/antmicro/hal_quicklogic/tree/initial_release))
* Run `west update`

I couldn't find any examples of storing bitstreams in Zephyr mainline, so I suppose this could be a good start for now. Anyway you can always extract submodule with IPs from `hal_quicklogic` and make it an independent Zephyr module in future.